_docs/110_faq.md

```diff
@@ -45,15 +34,6 @@ yadm gitconfig "user.name"  "Alternate Name"
 Note: Configuring these settings directly in the yadm repo will require you
 to configure it each time you clone the repo.
 
-### Why was I prompted to login twice while cloning my repo?
```

Comment:
Why remove this section? It is still relevant.

Reply:
The new version of clone doesn't need to inspect the default branch from the remote anymore. So the possible double login experience is gone.